### PR TITLE
Keep old ifcfg content

### LINKED ISSUE #99: ifcfg-* files are completely rewritten
When you change an interface configuration, the `ifcfg-*` gets completely overwritten. The problem is that only the values that the module know about are written. So comments and any other unknown key is just removed from the file.

### PR DESCRIPTION
**Pull request by imobachgs**
Fixes #99. Now any unknown key and comments are preserved.

### CODE REVIEW

File: src/lib/wicked/files.test.js [hunk 1]
```diff
@@ -173,13 +218,13 @@ describe('SysconfigFile', () => {
 
         it('returns the value for the given key', async () => {
             await file.read();
-            expect(file.get('NAME')).toBeUndefined();
-            expect(file.get('BOOTPROTO')).toEqual('dhcp');
-            expect(file.get('STARTMODE')).toBeUndefined();
+            expect(file.getKey('NAME')).toBeUndefined();
+            expect(file.getKey('BOOTPROTO')).toEqual('dhcp');
+            expect(file.getKey('STARTMODE')).toBeUndefined();
```
Comment (dgdavid) [inline src/lib/wicked/files.test.js:223]: I miss around here a test for the #getKey with a default value `file.getKey('STARTMODE', 'boot')`

**Review by dgdavid — APPROVED** [review]
LGTM, only a small suggestion.

File: src/lib/wicked/files.test.js (hunk 1 above)
Comment (imobachgs) [reply src/lib/wicked/files.test.js:223]: Good point. I have addded a test for that. Thanks!

**Review by dgdavid — APPROVED** [review]

**Review by teclator — APPROVED** [review]
LGTM

**Pull request merged**